Models/Database/Sighting.cs

```diff
@@ -9,14 +10,31 @@ public class Sighting
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
+        public string ApiId { get; set; }
         public string Species { get; set; }
-        public int Quantity { get; set; }
+        public string Quantity { get; set; }
```

Comment:
This looks a little bit surprising - is this really a string?

Reply:
yep! because "20+" is a valid entry (annoyingly)